libsplinter/Cargo.toml

```diff
@@ -36,7 +36,7 @@ actix-web-actors = { version = "1.0", optional = true }
 actix-web-3 = { package = "actix-web", version = "3", optional = true, features = ["openssl"] }
 atomicwrites = "0.2"
 awc = { version = "0.2", optional = true }
-base64 = { version = "0.12", optional = true }
+base64 = "0.12"
```

Comment:
Can this remain optional and move to a dependency of the "oauth" feature?

Reply:
Updated